### ISSUE TITLE
Migration error when using CockroachDB

### ISSUE THREAD

**Issue by tlvenn** [elixir-error-tracker/error-tracker#43]
When trying to run the migration with Postgrex & CockroachDB, the following error occurs:
```
[info] execute "INSERT INTO public.error_tracker_meta (key, value)\nVALUES ('migration_version', '2'), ('migration_timestamp', 1723522363)\nON CONFLICT (key) DO UPDATE SET value = EXCLUDED.value\n"
** (Postgrex.Error) ERROR 42804 (datatype_mismatch) VALUES types int and string cannot be matched

    query: INSERT INTO public.error_tracker_meta (key, value)
VALUES ('migration_version', '2'), ('migration_timestamp', 1723522363)
ON CONFLICT (key) DO UPDATE SET value = EXCLUDED.value
```

I am wondering if we could rewrite it following this pattern, it will probably fix the issue.
```
INSERT INTO t (a, b, ..., n)
VALUES ('1', '2', ..., 'n')
ON CONFLICT DO UPDATE SET
  a = '1',
  b = '2',
  ...
  n = 'n';
```

**Comment by crbelaus** [comment]
Thanks for reporting this. We are trying to save the timestamp as an integer. It looks like PostgreSQL is converting it to string implicitly but CockroachDB does not.

We should save it as a string explicitly for consistency with the table definition. This should fix the problem.

**Comment by crbelaus** [comment]
This should be fixed on #44. @tlvenn can you check if it works for you as well?

**Comment by tlvenn** [comment]
Working fine now, thanks for the quick fix !
